ucsb-cs156-f24/proj-happycows-f24-10
# Create course controller with get all endpoint and tests

## Overview
Add a course controller with the ability to get all as well as tests for get all.
Dokku Deployment (database is still empty): https://happycows-dev-r0ckwav3.dokku-10.cs.ucsb.edu/

## Validation 

1. Go to the dokku deployment's swagger
2. Verify that the get all endpoint exists

Or

1. Download the branch locally
2. Run `mvn test`

## Tests


## Linked Issues

Closes #34
